src/osuTK/Input/FileDropEventArgs.cs

```diff
@@ -14,7 +14,7 @@ public class FileDropEventArgs : EventArgs
         public string FileName
         {
             get;
-            internal set;
+            set;
```

Comment:
why is this required?

Reply:
This was still a part of my incorrect understanding of the eventhandler. I have reverted this change.

Reply:
doesn't look to be reverted.

Reply:
See the edit in my comment below as to why it wasn't reverted: 

> Edit 2: Seems I do require "internal set" -> "set"
> `error CS0200: Property or indexer 'FileDropEventArgs.FileName' cannot be assigned to -- it is read only [C:\projects\opentk\src\osuTK.iOS\osuTK.iOS.csproj]` Good of Appveyor to catch that cause somehow building it in MSVC didn't throw this (probably still built it from the cache despite the change)
> 
> It is needed due to osuTK.NS20 being in a different assembly from osuTK.iOS, so that kind of makes sense.

FileDropEventArgs also inherits from EventArgs so if you want to keep it `internal` that + FileDropEventArgs have to be recreated in the osuTK.iOS project